openpype/hooks/pre_add_last_workfile_arg.py

```diff
@@ -23,7 +25,7 @@ class AddLastWorkfileToLaunchArgs(PreLaunchHook):
     ]
 
     def execute(self):
-        if not self.data.get("start_last_workfile"):
+        if not self.data.get("start_last_workfile", True):
```

Comment:
Why was the `True` added as default value? If key `"start_last_workfile"` is not defined then should skip the logic?